src/cli/run.rs

```diff
@@ -49,6 +50,15 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     // Load the project
     let project = Project::load_or_else_discover(args.manifest_path.as_deref())?;
 
+    // Sanity check of prefix location
+    verify_prefix_location_unchanged(
+        project
+            .default_environment()
+            .dir()
+            .join(consts::PREFIX_FILE_NAME)
+            .as_path(),
+    )?;
```

Comment:
The default environment might not be created at this point. (Since you never run the default environment). Is that a problem?